Source/Public/Update-VSTeamTaskGroup.ps1

```diff
@@ -0,0 +1,31 @@
+function Update-VSTeamTaskGroup {
+   [CmdletBinding()]
```

Comment:
Add SupportsShouldProcess = $true and a ConfirmImpact value. You can review Update-VSTeamBuildDefinition for an example.